spec/models/hearing_spec.rb

```diff
@@ -51,6 +51,7 @@
 
       it { expect(hearing.judge_names).to eq(['Mr Recorder J Patterson']) }
       it { expect(hearing.prosecution_advocate_names).to eq(['John Rob']) }
+      it { expect(hearing.defence_advocate_names).to eq(['John Rob']) }
```

Comment:
Just a nitpick: it would be ideal to have a different name than the `prosecution_advocate_names`